src/libsystemd-network/radv-internal.h

```diff
@@ -65,6 +67,11 @@
 /* Pref64 option type (RFC8781, section 4) */
 #define RADV_OPT_PREF64                           38
 
+/* rfc6275 7.4 Neighbor Discovery Home Agent Lifetime (0-65520).
```

Comment:
Still I think the maximum should be 65535, but with short comment that the RFC is ambiguous (Maybe, the same comment should be added to the man page).
If a host refuse lifetime larger than 65520, the admin of the router can still set this value smaller than or equal to 65520.

Reply:
https://github.com/gzenux/asuswrt-rtn18u/blob/e44200cfa6b27d6db5b8073e0943c9d9d123db8c/release/src/router/quagga/zebra/interface.h#L165 . I dont see any one use 65535

Reply:
Other implementations are not relevant here.
The only important thing here is that the RFC is simply ambiguous.
Even if we can send larger value, there should be no problem, as admin can always pick a shorter value.

Reply:
I am not against it adding but I do not find any implementations like others do.

Reply:
I've found one entity who don't use `65535` as the maximum, but strangely, [they use `65534`](https://www.arubanetworks.com/techdocs/ArubaOS_8.8.0_Web_Help/Content/arubaos-solutions/mobility/conf-adva-mobi-func.htm); no explanation given.

Reply:
Yeah ```40-65534``` but why 40 ?

Reply:
65535 is typically used for infinity. So the implementation do not want to use 65535, I guess.
But, the RFC does not say that 65535 should be handled as infinity. The RFC is quite ambiguous.

I do not find the source of 40, but yeah too short lifetime is bogus. But, that's also admin's responsibility, and I do not think we should refuse such too short lifetime, unless RFC explicitly says.